service/templates/perfdiscovery.html

```diff
@@ -0,0 +1,10 @@
+{{define "scripts"}}
+<script type="text/javascript" src="/static/app/perfdiscovery/PerformanceDiscoveryCtrl.js?hash={{ StaticsMD5 }}"></script>
```

Comment:
Oh, the file doesn't exist in this branch. I think it should be ok, because the page is empty anyway. My next PR will fix it.